Docs/Debugging/Debugging with Eclipse and pyOCD.md

```diff
@@ -0,0 +1,68 @@
+
+##Initial Setup
+1. Download the latest version of eclipse with CDT.
+1. Install the GNU ARM Eclipse plugin.
+	1. Open Eclipse.
+	1. Click the **Help** menu item and select **Install New Software**.
+	1. In the **Work with** box paste the install address and press enter - [http://redmine.laoslaser.org/boards/3/topics/635](http://sourceforge.net/projects/gnuarmeclipse/files/Eclipse/updates/).
+  1. The package **GNU ARM C/C++ Cross Development Tools** should appear. Select it.
+  1. Click **Next** repeatedly and accept licence agreements.
+  1. Click **Finish**. If prompted to restart Eclipse, select **Yes**.
+1. [Install tools necessary for makefile projects](https://developer.mbed.org/handbook/Exporting-to-Make):
+  1. [GCC ARM Embedded](https://launchpad.net/gcc-arm-embedded).
+  1. Windows only
+    1. Add GCC ARM Embedded bin directory to path after installing - Ex. C:\Program Files (x86)\GNU Tools ARM Embedded\4.9 2015q2\bin.
+    1. Install **make** and add it to path. Make is available as one of the tools here: [http://gnuwin32.sourceforge.net/](http://gnuwin32.sourceforge.net/).
+1. Install python 2.7.9 or above
+  1. Windows Only - Make sure to select "Add to path" when installing.
+1. Install pyOCD by running "pip install pyocd".
+
+##Importing to Eclipse and building
+1. In the mbed online compiler, export the project you want to debug as **GCC (Arm Embedded)**.
+1. Extract the project archive exported from mbed.
+1. Open Eclipse.
+1. Click the **File** menu item, mouse over **New** and select **Makefile Project with Existing Code**.
+  1. Paste the location of your project into **Existing Code Location**.
+  1. **Project Name** should automatically populate. Update name here if desired.
+  1. In **Toolchain for Indexer Settings**, select **Cross ARM GCC**.
+  1. Click **Finish** to create the project.
+1. To set up building in Eclipse, go to the **Project** menu item, and select **Properties**.
+  1. Select **C/C++ Build**.
+  1. Unselect **Use default build command**.
+  1. For **Build command**, enter **make DEBUG=1**. The option "DEBUG=1" configures the make file to build optimized code, which makes debugging easier.
+  1. Click **Apply**, and then click **OK**.
+  <span class="images">![](Images/30_-_build_config.png)</span>
+1. Click the build button in Eclipse, and verify the project builds successfully.
+
+##Setting up debug configuration
+The last step before actually debugging is to set up the debug configuration. This only needs to be done once for per project. Once created, the configuration can be checked into revision control, allowing anyone else working on the project to use it without needing to go through setup again.
+
+1. On the **Run** menu item, select **Debug Configurations...** to bring up configurations.
```

Comment:
We don't use "On" for menu items. This can probably be Select **Run** > **Debug Configurations...**
"Bring up" is the wrong verb, too. It should just be "open" and then you need to know if it's a window or dialog box. You can just go with "To change configurations". It should come before the action (why would I perform an action if I don't know what it does?)